README.md

```diff
@@ -400,6 +401,36 @@ MaintenanceTasks.ticker_delay = 2.seconds
 
 If no value is specified, it will default to 1 second.
 
+#### Customizing which Active Storage service to use
+
+The Active Storage framework in Rails 6.1 and up supports multiple storage
+services per environment. To specify which service to use,
+`MaintenanceTasks.active_storage_service` can be configured with the service's
+key, as specified in your application's `config/storage.yml`:
+
+```yaml
+# config/storage.yml
+gcs_primary:
+  service: GCS
+  credentials: <%= Rails.root.join("path/to/primary/keyfile.json") %>
+  project: "my-project"
+  bucket: "primary-bucket"
+
+gcs_secondary:
```

Comment:
Maybe we could use more realistic names like `user_data` and `internal` or something like that?